robocop/utils/misc.py

```diff
@@ -182,7 +182,7 @@ def find_similar(self, name, candidates):
         if not matches:
             return ""
         matches = self.get_original_candidates(matches, norm_cand)
-        suggestion = " Did you mean:\n"
```

Comment:
Is it okay? It's displayed together with error message. For example:
Did not found X. Did you mean?

So missing space will result in ``Did not found X.Did you mean?``

Reply:
Ah nevermind, I see you've updated the space in other place.